product.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="src/main.css">
+    <title>Enter Your Pokemon</title>
+</head>
+<body>
+    <main>
+        <!-- section for pokmone info and h2 for name and type? -->
+        <section>
+            <h2>About Your Pokemon</h2>
+            <p>
```

Comment:
try not to use `<p>` for this. You could use a section/article and then set the display to block. Try out different CSS features to get the style you desire.